rkt/fetch_keys.go

```diff
@@ -0,0 +1,237 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/appc/spec/discovery"
+	"github.com/coreos/rkt/Godeps/_workspace/src/golang.org/x/crypto/openpgp"
+)
+
+// getPubKeyLocations either returns the locations supplied in argv or discovers one at prefix
+func getPubKeyLocations(prefix string, allowHTTP bool, debug bool) ([]string, error) {
+	if prefix == "" {
+		return nil, fmt.Errorf("at least one key or --prefix required")
+	}
+
+	kls, err := metaDiscoverPubKeyLocations(prefix, allowHTTP, debug)
+	if err != nil {
+		return nil, fmt.Errorf("--prefix meta discovery error: %v", err)
```

Comment:
Ditto, I guess. The `--prefix` here is confusing. Maybe just remove the leading slashes?